nep-3.mediawiki

```diff
@@ -129,6 +129,9 @@ ParameterType enum has the following values:
 | Array
 | An array of objects. The type of elements can be any value of ParameterType.
 |-
+| Map
+| A map of objects. The type of elements inside the key/value collection can be any value of ParameterType.
```

Comment:
@shargon I don't think this description is correct.
* A `Map` only takes a PrimitiveType as key (Boolean, ByteArray, Integer)
* A `Map` takes `StackItem` as a value. `Any` and `Void` are not `StackItem`'s.

src: https://github.com/neo-project/neo-vm/blob/754bca3568c50695c571cb796c500fd66c07d0fe/src/neo-vm/Types/Map.cs#L13

Reply:
Bullet 2 also holds for the `Array` description above it.

Reply:
Please, could you change it?